hadoop-ozone/dist/src/main/smoketest/ozonefs/hadoopo3fs.robot

```diff
@@ -33,8 +33,11 @@ Test hadoop dfs
     ${dir} =          Format FS URL         ${SCHEME}     ${volume}    ${bucket}
     ${random} =        Generate Random String  5  [NUMBERS]
     ${result} =        Execute                    hdfs dfs -put /opt/hadoop/NOTICE.txt ${dir}/${PREFIX}-${random}
+                       Log to console             ${result}
                        Should Not Contain         ${result}           multiple SLF4J bindings
     ${result} =        Execute                    hdfs dfs -ls ${dir}
+                       Log to console             ${result}
                        Should contain             ${result}   ${PREFIX}-${random}
     ${result} =        Execute                    hdfs dfs -cat ${dir}/${PREFIX}-${random}
+                       Log to console             ${result}
```

Comment:
Please remove these `Log to console` statements.  I don't think they are related to the PR, but most importantly: they clutter the output unnecessarily.

https://github.com/vtutrinov/ozone/actions/runs/6534482769/job/17742733222#step:5:184

Reply:
Done